edbdeploy/cloud.py

```diff
@@ -175,6 +175,45 @@ def check_instances_availability(self, region):
             )
 
 
+class AWSRDSCli(AWSCli):
+    def check_instance_type_availability(self, instance_type, region):
+        try:
+            output = exec_shell([
+                self.bin("aws"),
+                "rds",
+                "describe-reserved-db-instances-offerings",
+                "--product-description postgresql",
+                "--region %s" % region,
+                "--output json"
```

Comment:
This command line does not look to check the availability of a particular instance type availability.
I think we should use: `aws rds describe-reserved-db-instances-offerings --product-description postgresql --region us-east-1 --output json --db-instance-class <instance_type>`

Reply:
I think I fixed this up in aee164a0fff7e6727eff0e7a60e4552a93606306